README.md

```diff
@@ -144,6 +144,18 @@ recommended way is to run ci/ci.sh. It will create a Python virtualenv with
 conan, setup the configuration for cross-compilation, register some recipes,
 and then run cmake and build pkgj for your vita and pkgj_cli for testing.
 
+Prerequisites:
+
+*  A working installation of VITASDK - full setup instructions can be found at https://vitasdk.org/.
```

Comment:
You don't need one, it's installed by conan

Reply:
Awesome - I didn't realise that.